gRPC/src/main/proto/cardCompany.proto

```
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package com.sub.grpc;
```

Buf (1.58.0)

Files with package "com.sub.grpc" must be within a directory "com/sub/grpc" relative to root but were in directory "gRPC/src/main/proto".

(PACKAGE_DIRECTORY_MATCH)